### PR TITLE
Fix InterfaceOverview to properly close `<dl>`s

### PR DESCRIPTION
**Pull request by a2sheppy**
Also removed the Experimental badge and merged the
Obsolete badge with the Deprecated one.

### CODE REVIEW

File: macros/InterfaceOverview.ejs [hunk 1]
```diff
@@ -260,18 +268,12 @@ if (slug) {
 
             // Add badges. We need them, seriously.
 
-            if (hasTag(aPage, 'Experimental')) {
-                result += " " + badges.ExperimentalBadge;
-            }
```
Comment (ExE-Boss) [inline macros/InterfaceOverview.ejs]: I don’t think we should be removing these in this PR.

File: macros/InterfaceOverview.ejs [hunk 2]
```diff
@@ -260,18 +268,12 @@ if (slug) {
...
             if (hasTag(aPage, 'Non-standard') || hasTag(aPage, 'Non Standard')) {
                 result += " " + badges.NonStandardBadge;
             }
-            if (hasTag(aPage, 'Deprecated')) {
+            if (hasTag(aPage, 'Deprecated') || hasTag(aPage, 'Obsolete')) {
                 result += " " + badges.DeprecatedBadge;
             }
-            if (hasTag(aPage, 'Obsolete')) {
-                result += " " + badges.ObsoleteBadge;
-            }
```
Comment (ExE-Boss) [inline macros/InterfaceOverview.ejs]: ditto

**Review by ExE-Boss — CHANGES_REQUESTED** [review]

**Review by ExE-Boss — CHANGES_REQUESTED** [review]

**Review by ExE-Boss — APPROVED** [review]
LGTM, r+

**Comment by a2sheppy** [comment]
Still awaiting review approval, fwiw.

File: macros/InterfaceOverview.ejs [hunk 3]
```diff
@@ -261,7 +269,7 @@ if (slug) {
             // Add badges. We need them, seriously.
 
             if (hasTag(aPage, 'Experimental')) {
-                result += " " + badges.ExperimentalBadge;
+               result += " " + badges.ExperimentalBadge;
```
Comment (ExE-Boss) [inline macros/InterfaceOverview.ejs]: Wrong indent.

**Comment by ExE-Boss** [comment]
review?(@jwhitlock)

**Comment by ExE-Boss** [comment]
review?(@wbamberg, @groovecoder)

**Comment by wbamberg** [comment]
I would like to see a failing test for this macro that is fixed by this change. Also, I think this PR should *only* fix this problem, and any other changes should be made in separate PRs (that should also add tests for the changes they make).

Also we should be cautious about investing a lot of time in macros that we might intend to retire. It's probably worth fixing this one though since AIUI it does currently break pages.

**Comment by a2sheppy** [comment]
Ready for a re-review and hopefully a merge.

**Comment by ExE-Boss** [comment]
This might be best merged using the <q>Squash and Merge</q> strategy.

**Comment by a2sheppy** [comment]
I sure hope we're not planning to retire this one, as I'm actively working on updates so we can use it more, not less... :)

**Comment by ExE-Boss** [comment]
I&nbsp;have written tests for&nbsp;this in&nbsp;https://github.com/mdn/kumascript/pull/1073.

**Pull request closed without merging**